### PR TITLE
mgr/dashboard: Cephfs subvolumegroup edit

### PR DESCRIPTION
**Pull request by Pegonzal**
Fixes: https://tracker.ceph.com/issues/62370

https://github.com/ceph/ceph/assets/106810838/c1b2bcf5-b9b1-43fa-86ce-aba32b80a3bc

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.ts [hunk 1]
```diff
@@ -101,35 +116,79 @@ export class CephfsSubvolumegroupFormComponent implements OnInit {
     });
   }
 
+  populateForm() {
+    this.action = this.actionLabels.EDIT;
+    console.log('populating form');
+    console.log(this.fsName);
```
Comment (aaSharma14) [inline src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.ts:122]: log statements can be removed

**Review by aaSharma14 — APPROVED** [review]
can we edit the sub-volume group permissions as well? if yes, the changes are not reflecting when I tried locally!

**Comment by nizamial09** [comment]
> can we edit the sub-volume group permissions as well? if yes, the changes are not reflecting when I tried locally!

we can't edit them. its disabled in the edit form when I tried it!

File: src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.ts (hunk 1 above)
Comment (nizamial09) [reply src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolumegroup-form/cephfs-subvolumegroup-form.component.ts]: removed

**Review by nizamial09 — APPROVED** [review]

**Review by cloudbehl — APPROVED** [review]
LGTM

**Comment by nizamial09** [comment]
dashboard e2e failure is because of a broken rgw-buckets failure.

**Pull request merged**